core/backend/src/BackendHubAccess.ts

```diff
@@ -164,7 +164,7 @@ export interface BackendHubAccess {
   queryChangeset(arg: ChangesetArg): Promise<ChangesetProps>;
   /** Query an array of changeset properties given a range of ChangesetIndexes  */
   queryChangesets(arg: ChangesetRangeArg): Promise<ChangesetProps[]>;
-  /** push a changeset to iMOdelHub. Returns the newly pushed changeSet's index */
+  /** push a changeset to iModelHub. Returns the newly pushed changeSet's index */

```

Comment:
changeSet's -> changeset's

Also, the documentation in V2CheckpointAccessProps is wrong/incomplete (apparently GitHub won't let me comment on a line you didn't change).